### PR TITLE
nvidia_x11_beta: 455.45.01 -> 460.27.04

### PR DESCRIPTION
**Pull request by Atemu**
###### Motivation for this change
https://github.com/NixOS/nixpkgs/pull/106999
https://github.com/NixOS/nixpkgs/pull/107105

Closes https://github.com/NixOS/nixpkgs/pull/107170

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by Atemu** [comment]
/marvin opt-in
/status needs_reviewer

cc @TimothyKlim

File: pkgs/os-specific/linux/nvidia-x11/default.nix [hunk 1]
```diff
@@ -30,7 +30,12 @@ rec {
     else legacy_390;
 
   # No active beta right now
```
Comment (danieldk) [inline pkgs/os-specific/linux/nvidia-x11/default.nix:32]: Could you remove this comment, since it is not relevant now?

**Review by danieldk — CHANGES_REQUESTED** [review]

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 107150` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>17 packages built:</summary>
  <ul>
    <li>linuxPackages-libre.nvidia_x11_beta</li>
    <li>linuxPackages.nvidia_x11_beta</li>
    <li>linuxPackages_4_14.nvidia_x11_beta</li>
    <li>linuxPackages_4_19.nvidia_x11_beta</li>
    <li>linuxPackages_4_4.nvidia_x11_beta</li>
    <li>linuxPackages_4_9.nvidia_x11_beta</li>
    <li>linuxPackages_5_10.nvidia_x11_beta</li>
    <li>linuxPackages_5_9.nvidia_x11_beta</li>
    <li>linuxPackages_hardened.nvidia_x11_beta</li>
    <li>linuxPackages_latest-libre.nvidia_x11_beta</li>
    <li>linuxPackages_latest_hardened.nvidia_x11_beta</li>
    <li>linuxPackages_latest_xen_dom0.nvidia_x11_beta</li>
    <li>linuxPackages_latest_xen_dom0_hardened.nvidia_x11_beta</li>
    <li>linuxPackages_testing_bcachefs.nvidia_x11_beta</li>
    <li>linuxPackages_xen_dom0.nvidia_x11_beta</li>
    <li>linuxPackages_xen_dom0_hardened.nvidia_x11_beta</li>
    <li>linuxPackages_zen.nvidia_x11_beta</li>
  </ul>
</details>

File: pkgs/os-specific/linux/nvidia-x11/default.nix (hunk 1 above)
Comment (Atemu) [reply pkgs/os-specific/linux/nvidia-x11/default.nix]: Oh yeah, right, lol ^^

Done.

**Review by danieldk — APPROVED** [review]
LGTM, thanks!

**Pull request merged**